examples/dll-entry-only/webpack.config.js

```diff
@@ -0,0 +1,24 @@
+var path = require("path");
+var webpack = require("../../");
+
+module.exports = {
+	// mode: "development || "production",
+	entry: {
+		dll: ["./example"]
+	},
+	output: {
+		path: path.join(__dirname, "dist"),
+		filename: "[name].js",
+		library: "[name]_[hash]"
+	},
+	optimization: {
+		concatenateModules: true
```

Comment:
Add a comment that this is only for example demonstration and not needed in a real config since it's including in `mode: "production"`